### PR TITLE
Add basic circleci config

### PR DESCRIPTION
**Pull request by rtburns-jpl**
There are no tests yet, so this currently just builds the sdist/wheel

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          name: "Install build system dependencies"
+          command: |
+            sudo apt-get update
+            sudo apt-get install python3 python3-setuptools python3-pip
+            python3 -m pip install --user build
+      - checkout
+      - run:
+          name: "Build COMPASS python package"
+          command: |
+            python3 -m build --sdist --wheel .
```
Comment (hfattahi) [inline .circleci/config.yml:19]: Thanks @rtburns-jpl. Compass depends on isce3 . The rest of dependencies are [here](https://github.com/opera-adt/COMPASS/blob/main/requirements.txt). I think if we add isce3 the rest of dependencies come with it. As we discussed we want to move forward assuming that compass works with isce3 on conda package.

Comment (LiangJYu) [reply .circleci/config.yml:19]: COMPASS also needs [s1-reader](https://github.com/opera-adt/s1-reader).

Comment (vbrancat) [reply .circleci/config.yml:19]: @rtburns-jpl any update on this PR?

Comment (rtburns-jpl) [reply .circleci/config.yml:19]: Currently focused on getting https://github.com/opera-adt/COMPASS/pull/30 merged - once we do so, this PR can reuse the docker build scripts for the CI runs.

Comment (vbrancat) [reply .circleci/config.yml:19]: #30 is merged, @rtburns-jpl you can go ahead and merge this PR if you are happy with it.

**Pull request merged**